forwards_paper.tex

```diff
@@ -915,41 +890,38 @@ \subsubsection*{Estimates of run-time complexity}
 
 
 %% simplify run time
-Above we have described memory requirements;
-how does the computation time required for simplification scale?
+How does the computation \emph{time} required for simplification scale?
 Simply because it must process each edge,
 % below per discussion
 % https://github.com/petrelharp/ftprime_ms/pull/37#discussion_r156617283
 the simplification algorithm is at least linear in the number of edges of the input tree sequence.
 Empirically the algorithm is exactly linear,
-as seen in Figure~\ref{fig:simplify_complexity}X,
+as seen in Figure~\ref{fig:simplify_complexity}B,
 where we show shows the time required to simplify increasingly large subsets of a large tree sequence.
 When simplifying the result of a forwards-time sequence, the number of edges is the main contributing factor.
 Suppose on the other hand we want to
 simplify an already-minimal but large tree sequence with $N$ nodes
 to a subsample of size $n$.
 How does the required time scale with $n$?
 In this case, the computation is dominated by the size of the output tree sequence,
-which grows with $\log(n)$.
-this is also seen in Figure~\ref{fig:simplify_complexity}X,
-showing the time required to simplify progressively larger subsets of a single large tree sequence.
-
-
-Our method stores genealogies, and so records substantially more information
-than would a method only recording genotypes.
-However, since the simplification algorithm requires computational effort,
-it is informative to compare computational complexity of the algorithm
-to one that propagates neutral genotypes.
-A typical individual differs at around $2 N \mu$ sites from the population's consensus sequence,
-so propagating these to offspring by simple copying will take $4 N^2 \mu$ operations per generation.
-On the other hand,
-in our scheme we must store $13N$ quantities per generation (two edges and one node per individual).
-\plr{XXX}
-The simplification algorithm is linear in the number of edges of the input tree sequence
-(because it must process each edge),
-and so multiplies this by a constant factor.
-These considerations imply that propagating neutral genotypes for $T$ generations has time complexity $O(\mu T N^2)$,
-while our implementation is only $O(T N \log(2N))$.
+which grows with $\log(n)$, as shown in Figure~\ref{fig:simplify_complexity}C,
+
+
```

Comment:
This paragraph below doesn't have much content: it compares to the naive method, which no-one good uses, and doesn't justify things.